team-validator.js

```diff
@@ -819,6 +820,9 @@ class Validator {
 							// must be able to breed with father
 							if (!dexEntry.eggGroups.some(eggGroup => eggGroupsSet.has(eggGroup))) continue;
 
+							const moveGen = (dexEntry.learnset[moveid] ? dexEntry.learnset[moveid][0].charAt(0) : dexEntry.learnset['sketch'][0].charAt(0));
+							if (noPastGenBreeding && (dexEntry.tier.startsWith('Bank') || moveGen !== '7' || template.id === dexEntry.prevo || dexEntry.evos.indexOf(template.id) >= 0)) continue;
```

Comment:
I see why I misread it. It looks like you're just looking at the `moveGen` for the first move. Which is, I guess, an acceptable hack......